hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -34,4 +39,27 @@ void HologramDataSourceAvailabilityFetcher::
                                                 &hologram_configs_));
 }
 
+void HologramDataSourceAvailabilityFetcher::FetchFromDatabase() {
+    // TODO(alexanderlin): Implement.
+}
+
+void HologramDataSourceAvailabilityFetcher::GetStatus(absl::CivilMinute date) {
+    // TODO(alexanderlin): Implement.
+    // for(const std::pair<std::string, std::string>& it : system_to_cell_map_) {
+    //     for(int i = 0; i < hologram_configs_.data_source_config_size(); ++i) {
+    //     UpdateProto(it.first, date, hologram_configs_.data_source_config(i), status);
+    // }
+}
+
+void HologramDataSourceAvailabilityFetcher::UpdateProto(std::string system, 
+    absl::CivilMinute date, DataSource data_source, StatusType status){
+    // TODO(alexanderlin) Implement.
+}
+
+void HologramDataSourceAvailabilityFetcher::UpdateHistory(
```

Comment:
Where is UpdateHistory got called?

Reply:
Added comments in UpdateProto to clarify where UpdateHistory is called